README.md

```diff
@@ -615,8 +615,12 @@ may also be specified using the options dict parameter.
 
 Possible `options`:
 
-* data_format - must be a string, choices: "dataframe", "records", "json"
+* data_format - must be a string, choices: "dataframe", "records", "json", "flattened_dataframe"
+  * "flattened_dataframe" is best used for JSON structure typically found in data streams that contain
+nested lists of dictionaries and a payload. For example: `{"data": [ columns ], "response": 200}`
 * selected_keys - columns being selected from the entire dataset, must be a list `["column 1", "ssn"]`
+* payload_key - The dictionary key for the payload of the JSON, typically called "data"
+or "payload". Defaults to "data".
```

Comment:
Should it be 'payload_keys' and set it as a list, e.g. should already be `["data", "response"]` at the very least